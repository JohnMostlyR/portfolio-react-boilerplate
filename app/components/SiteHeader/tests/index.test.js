import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SiteHeader from '../index';

describe('<SiteHeader />', () => {
  it('should render and match the snapshot', () => {
    const wrapper = shallow(<SiteHeader />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
