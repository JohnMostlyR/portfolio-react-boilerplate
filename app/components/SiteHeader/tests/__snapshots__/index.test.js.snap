// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteHeader /> should render and match the snapshot 1`] = `
<Header
  backgroundColor="#465a65"
>
  <Wrapper
    borderColor="#465a65"
  >
    <Brand
      href="/"
      showNameBreakpoint="375px"
    />
    <HeaderMenu />
  </Wrapper>
</Header>
`;
